ui_framework/components/modal/confirm_modal.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { KuiModal } from '../modal';
+import { KuiModalFooter } from './modal_footer';
+import { KuiModalHeader } from './modal_header';
+import { KuiModalBody } from './modal_body';
+import { KuiModalBodyText } from './modal_body_text';
```

Comment:
Ah, yes, I suppose it would. One more reason for me to dislike the root level index.js that exports *all* the modules then. 🏆